lib/data/README.md

```diff
@@ -1,9 +1,18 @@
 
 ## HUD Lists
 
-The HUD Lists JSON files are written by hand. They match the lists specified in the HUD HMIS CSV specifications.
+For a new year, copy the previous year_hud_list.json file and the year_additional_lists directory and then rename them with the new year suffix. Update files in the the year_additional_lists with data from the new specs.
 
-When making any changes to the JSON files, please run the following tasks to re-generate dependent code:
+Run the json generator `code:generate_hud_list_json year file_path_to_machinde_readable_xlsx_file`
+
+e.g.
+```
+rails code:generate_hud_list_json\["2024","lib/data/CSV Specifications Machine-Readable_FY2024.xlsx"\]
+```
+
+This will generate the JSON files. To account for some inconsistencies with the excel data, some names will be generated as "Unknown" and will need to be verified manually with the lists specified in the HUD HMIS CSV specifications.
+
+After changes to the JSON files have been confirmed, please run the following tasks to re-generate dependent code:
 
 ```
 rails code:generate_hud_lists
```

Comment:
Cool!!

Running this (and the line below it) shows the impact of this change. We do use the labels in the HMIS UI (as does the warehouse) so the casing changes will show up in the interface. Can you run those scripts and re-push @dtgreiner so we can review the effects of this change?

I need to take time to go through it more closely. A few items of concern jump out to me, though: 
* label change `Bed Night` => `BedNight`
* label changes where the apostrophe becomes a different type of apostrophe. specifically `Client doesn’t know` I want to avoid churn on the 1.8 list (yes/no/dk/r/dnc) because we use that one for custom data elements which store the string label value). would prefer to avoid this churn
* label changes where they got more verbose (relationships_to_hoh) - I can/will add overrides to the frontend if we want to keep it this way in the backend
* labels where n-dash is replaced with m-dash -- feels like unnecessary churn but is probably fine
* `record_types` label casing became lowercase
* labels where instructional parenthesized text is ok -- maybe fine, but I want to go through them

Reply:
Also hitting this -- I had separated out all the different living situation categories in hud_lists.json so we need to keep that somehow. (probably as additional_lists)

```
82c816eeaedb:/app$ rails driver:hmis:dump_graphql_schema
rails aborted!
NoMethodError: undefined method `destinations' for HudUtility2024:Module

    hud_enum HudUtility2024.destinations
                           ^^^^^^^^^^^^^
Did you mean?  destination_type
```

Reply:
I still get errors when running `rails driver:hmis:dump_graphql_schema` due to some missing enums. Sorry I'm assuming this probably wasn't part of the original scope of this work @dtgreiner @eanders. I'm happy to take on the graphql parts of things if we want to defer that, but we probalby won't schedule work for that for several weeks, and I don't love the idea of the schema being out-of-sync for so long from the source files. Maybe we can check in about this today?